.github/workflows/ssp-ci.yml

```diff
@@ -0,0 +1,32 @@
+name: Self-serve Portal CI
```

Comment:
nit: could we possibly call this file `ssportal-ci.yml`? my head went to Supply-Side-Platform for a sec 😆

Reply:
Yeah, makes sense, haha I will update the file name